src/components/20-molecules/top-content-bar/index.js

```diff
@@ -46,13 +55,31 @@ class AXATopContentBar extends InlineStyles {
     super();
     applyDefaults(this);
 
+    window.addEventListener('axa-top-bar-open', () => {
+      const contBarEl = this.shadowRoot.querySelector(elRootSelector);
+      if (contBarEl) {
+        contBarEl.classList.remove(hideClass);
+      } else {
+        elementHidden = false;
+      }
+    });
+
     defineVersioned(
-      [AXAButton, AXAButtonLink, AXAContainer],
+      [AXAButton, AXAButtonLink, AXAContainer, AXAIcon],
       __VERSION_INFO__,
       this
     );
   }
 
+  onClose() {
+    const contBarEl = this.shadowRoot.querySelector(elRootSelector);
+    contBarEl.style.maxHeight = 0;
+    setTimeout(() => {
+      contBarEl.classList.add(hideClass);
+      fireCustomEvent('axa-top-bar-close', null, window);
+    }, 300);
```

Comment:
why do we need a 300ms delay?

Reply:
@Ben-Zahler 
I tried to add some animation into closing process although that was not the requirement and also it doesn't work for desktop-overlay so I will just remove this timeout and do the simple close.